C-Plus-Plus/cp/Reverse_string_using_stack.cpp

```diff
@@ -0,0 +1,58 @@
+/* 
+Given a string,print reverse of the string using stack
+Approach:-
+In stack,we know LIFO principle works.
+First,i create an empty stack,then i push each character of string in stack
+then,using for loop i popped out every single character.
+*/
+
+#include<iostream>
+#include<string>
+#include<stack>
+using namespace std;
+
+//function to reverse the string
+void reverse(string &s)
+{
+	int i;
+	stack<int> str;
+
+	for(char ch:s)
+	{
+        str.push(ch);
+    	}
+    	for(i=0;i<s.size();i++)
+    	{
+        s[i]=str.top();
+        str.pop();
+    	} 
+}
+
+int main()
+{
+	cout<<"***********Program to reverse a string using stack***********"<<endl;
+	string s;
+	cout<<"Enter a string you want to reverse: ";
+	cin>>s;
+
+	reverse(s);
+    cout<<"String After Reverse: ";
+	cout<<s;
+	return 0;
+}
+
+/*
+Test Case 1:
+Input :
+***********Program to reverse a string using stack***********
+
```

Comment:
Remove that extra space.
![Capture](https://user-images.githubusercontent.com/67387053/111436810-694c9400-8728-11eb-939c-96663504cc0f.JPG)